shell/platform/windows/win32_window.cc

```diff
@@ -0,0 +1,207 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/win32_window.h"
+
+namespace flutter {
+
+Win32Window::Win32Window() {
+  dpi_helper_ = std::make_unique<Win32DpiHelper>();
+
+  // Assume Windows 10 1607 for DPI handling.
+  // TODO handle DPI robustly downlevel
+  // TODO the calling applicaiton should participate in setting the DPI
+  // awareness mode
+  auto result = dpi_helper_->SetProcessDpiAwarenessContext(
```

Comment:
Please only use `auto` when it increases readability (e.g., when the right hand side has the type such as a make_unique or cast, or complex iterator declarations), per style. Applies throughout this file, which uses auto very heavily.